rdbms/src/main/java/org/spine3/server/storage/jdbc/entity/visibility/table/VisibilityTable.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.server.storage.jdbc.entity.visibility.table;
+
+import org.spine3.server.entity.Visibility;
+import org.spine3.server.storage.jdbc.util.DbTableNameFactory;
+
+/**
+ * A container for the common constants for all the queries to the table storing
+ * the {@linkplain Visibility}.
+ *
+ * @author Dmytro Dashenkov.
+ */
+public class VisibilityTable {
```

Comment:
The constants do not belong to an `Abstract...Query` because
 - they are used not only in queries but also in utility classes;
 - even if not for that, the smallest common ancestor of all the queries using these values is `StorageQuery` - the most abstract representation of an SQL query in our library.

Also, note that there are more than one such classes. If eliminating one, we should propagate this policy to others. 
Why don't we do this in the next incrementation?